Feature/infolist item playground

ektasawant opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->
Fixes 3489 .

<!-- Include a bulleted list summarizing the main changes you have made in this PR -->
#### Changes proposed in this Pull Request:
- Playground for InfolistItem
- 
- 

<!-- Include screenshots if they will help illustrate the changes in this PR -->
#### Screenshots:
<img width="1785" alt="Screenshot 2022-09-12 at 4 28 27 PM" src="https://user-images.githubusercontent.com/10433274/189637100-c601eead-0e0b-404b-9026-013952150ebf.png">

<!-- Instruction for PR reviewers, if more complicated than a simple yarn start -->
#### To Test:
- http://localhost:3000/components/info-list-item/playground

emclaug2 commented:
It might be beneficial to add some sort of light border around the InfoListItem, or give it a background color by defaut so we can see where it starts & ends.

emclaug2 commented:
Code generation looking good; one thing I've noticed - can you remove the `info` & `subtitle` props when no value is provided?

![image](https://user-images.githubusercontent.com/6538289/189663401-28e38ebb-d98c-45f3-99c5-9d21d60eb624.png)

jeffvg commented:
should the top accordion in playground have a title? other than that looks good

![image](https://user-images.githubusercontent.com/37150565/189695533-406d2121-1fe3-4fb4-8cec-948716762f7a.png)

emclaug2 commented:
@ektasawant The changes look good; I've got the same question as Jeff:

![image](https://user-images.githubusercontent.com/6538289/189717116-529186a9-5ec0-41ae-8d0a-2e69b9b027f3.png)


Also one more thing, a style suggestion - I've added a max-width of 700px to the InfoListItem example since it looks a little weird if it stretches all the way across the screen.

https://blui-angular-docs--pr495-feature-3442-infolis-2blc8cfa.web.app/components/info-list-item/playground

ektasawant commented:
> https://blui-angular-docs--pr495-feature-3442-infolis-2blc8cfa.web.app/components/info-list-item/playground

- Added `required props` as a section name.
- Added max-width 700px

jeffvg reviewed (approved):
looks good

daileytj commented on docs/src/componentDocs/InfoListItem/playground/InfoListItemConfig.tsx:
```diff
@@ -0,0 +1,187 @@
+            inputType: 'boolean',
+            inputValue: false,
+            propType: 'boolean',
+            helperText: 'Smaller height row with less padding	',
```
```suggestion
            helperText: 'Smaller height row with less padding',
```

daileytj reviewed (approved):
Looks good to me

daileytj commented:
![Screen Shot 2022-09-13 at 9 21 28 AM](https://user-images.githubusercontent.com/13989985/189912860-bbe5dbda-dd18-4772-adcf-bcb081ed1fe8.png)
 WrapTitle, WrapSubtitle, and WrapInfo aren't working as expected. I thought this may be an issue with the component, but they work fine within the old storybook so I don't think this is a component issue.

ektasawant commented:
> https://blui-react-docs--pr512-feature-infolistitem-ai73aaik.web.app

<img width="1675" alt="Screenshot 2022-09-13 at 8 15 44 PM" src="https://user-images.githubusercontent.com/10433274/189932398-f5e3777e-25bf-44be-94f5-8213ee25a415.png">
I am getting this when checked WrapTitle, WrapSubtitle, and WrapInfo.

daileytj commented:
@ektasawant Looks fine. Must have been an issue with my local build. ¯\_(ツ)_/¯
